example/lib/dbpage_routing_example.dart

```diff
@@ -1,4 +1,6 @@
 import "package:dbpage_routing/dbpage_routing.dart";
 
 void main() {
+	print("random stup");
+	final unused_stuff = "dont worry";
```

Comment:
The value of the local variable 'unused_stuff' isn't used.
**SEVERITY**: ERROR
**RULE**: UNUSED_LOCAL_VARIABLE
**FILE**: lib/dbpage_routing_example.dart
**LINE**: 5